keepassxc-browser/background/page.js

```diff
@@ -321,10 +321,9 @@ page.retrieveCredentials = async function(tab, args = []) {
 
 page.getLoginId = async function(tab) {
     // If there's only one credential available and loginId is not set
-    if (page.tabs[tab.id] && page.tabs[tab.id].loginId < 0
-        && page.tabs[tab.id]
+    if (page.tabs[tab.id] && !page.tabs[tab.id].loginId
         && page.tabs[tab.id].credentials.length === 1) {
-        return 0; // Index to the first credential
+        return page.tabs[tab.id].credentials[0].uuid;
```

Comment:
for clarity in reading you should store the tab in a variable: `currentTab = page.tabs[tab.id]`